java/src/org/openqa/selenium/grid/node/docker/DockerFlags.java

```diff
@@ -69,6 +69,18 @@ public class DockerFlags implements HasRoles {
           "[\"selenium/standalone-firefox:latest\", \"{\\\"browserName\\\": \\\"firefox\\\"}\"]")
   private List<String> images2Capabilities;
 
+  @Parameter(
+      names = {"--docker-host-config-keys"},
+      description =
+          "Node container host config keys to be passed to the browsers container. Keys name can be"
+              + " found in the Docker API documentation, or by running `docker inspect` the"
+              + " node-docker container.")
+  @ConfigValue(
+      section = DockerOptions.DOCKER_SECTION,
+      name = "host-config-keys",
+      example = "[\"Dns\", \"DnsOptions\", \"DnsSearch\", \"ExtraHosts\"]")
+  private List<String> hostConfigKeys;
+
```

Comment:
We need to add this to the docs.

Reply:
Sure, I will add this to docs repo and also docker-selenium.